### PR TITLE
supress warning for memory calculator

### PR DESCRIPTION
**Pull request by oovk**
Issue:#176

## Summary
Fix issue related to memory calculator warning. See #176

## Use Cases
cgroups v2 without memory limit set

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by pivotal-david-osullivan** [comment]
Thank you for the contribution! You'll need to follow the instructions [above](https://github.com/paketo-buildpacks/libjvm/pull/180#issuecomment-1165436210) to get the EasyCLA bot happy before any PR can be merged, could you try this and re-push the commit?

**Comment by dmikusa-pivotal** [comment]
+1 changes look good, but we need to have the CLA signed before we can merge. Were you able to follow the steps on the link above? Or are you waiting on approval through your company? Thanks

**Comment by oovk** [comment]
@dmikusa-pivotal CLA  approved.:smile: :+1:

**Review by pivotal-david-osullivan — APPROVED** [review]

**Pull request merged**
